network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/article_image_block.html

```diff
@@ -0,0 +1,15 @@
+{% load wagtailimages_tags wagtailcore_tags remove_richtext_wrapping %}
+
+{% comment %}This block doesn't need to "break" out of the Publications container{% endcomment %}
+<div class="article-image-block">
+  {% comment %}Use a 2x resolution image{% endcomment %}
+  {% image self.image width-1840 as img %}
+  <figure class="my-default">
+    <img src="{{ img.url }}" alt="{% if self.alt_text %}{{ self.alt_text }}{% else %}{{ img.alt }}{% endif %}" class="w-100" />
+    {% if self.caption %}
+        <figurecaption>
+          {{ self.caption|richtext|remove_wrapping|safe }}
+        </figurecaption>
```

Comment:
```suggestion
        <figcaption>
          {{ self.caption|richtext|remove_wrapping|safe }}
        </figcaption>
```

Reply:
LOL that's funny this was never caught until now. Good catch Mavis!